### ISSUE TITLE
Kedro Viz Build 

### ISSUE THREAD

**Issue by amandakys** [kedro-org/kedro-viz#1615]
## Description
We want to provide the `kedro viz build` command to give users access to a build directory that they can then deploy using any static site hosting provider of their choice. 

The 'backend' of this work is already done to enable the AWS focused Hosted Viz feature. It works by 
1. assembling all the necessary build files and generating static data files 
2. sending this to a s3 bucket so it can be hosted as a static site.  

For this feature, we would interrupt after step 1, and return the build files in a /build directory inside the project. 

## Acceptance Criteria 
- [ ] working `kedro viz build` command that creates the build files in the `/build` directory 
- [ ] running `kedro viz build` again will **overwrite** the existing contents of `/build` 
- [ ] if a `/build` directory doesn't already exist, create one 
- [ ] update the `kedro viz` help text to include the `build` command 
 `build       Create build directory of local Kedro Viz instance with static data `
